src/category_theory/monad/adjunction.lean

```diff
@@ -54,16 +61,55 @@ def comparison_forget [is_right_adjoint R] : comparison R ⋙ forget (left_adjoi
 
 end monad
 
-/-- A right adjoint functor `R : D ⥤ C` is *monadic* if the comparison function `monad.comparison R` from `D` to the
-category of Eilenberg-Moore algebras for the adjunction is an equivalence. -/
+namespace comonad
+
+/--
+Gven any adjunction `L ⊣ R`, there is a comparison functor `category_theory.monad.comparison R`
+sending objects `Y : D` to Eilenberg-Moore algebras for `L ⋙ R` with underlying object `R.obj X`.
+
+We later show that this is full when `R` is full, faithful when `R` is faithful,
+and essentially surjective when `R` is reflective.
+-/
+@[simps]
+def comparison [is_left_adjoint L] : C ⥤ coalgebra (right_adjoint L ⋙ L) :=
```

Comment:
The docstring and the def seem out of sync (dual?). Also below.

Reply:
You're completely right, it was a lazy copy and paste on my part